src/Analyzer.Reports/ConsoleReportWriter.cs

```diff
@@ -39,6 +54,7 @@ private void OutputToConsole(IEnumerable<Types.IEvaluation> evaluations)
                 {
                     string resultString = GenerateResultString(evaluation);
                     var output = $"{IndentedNewLine}{(evaluation.RuleId != "" ? $"{evaluation.RuleId}: " : "")}{evaluation.RuleDescription}" +
+                        $"{TwiceIndentedNewLine}Severity: {evaluation.Severity}" + 
```

Comment:
Match indentation here - either move it out to the same level as others, or indent others as well.